gulpfile.coffee

```diff
@@ -80,5 +76,8 @@ gulp.task 'pack-browser', ->
 		.pipe(minify())
 		.pipe(gulp.dest(OPTIONS.directories.build))
 
+gulp.task 'build',
+	gulp.series(['build-node', 'pack-browser'])
+
 gulp.task 'build-browser', (callback) ->
```

Comment:
```suggestion
gulp.task 'build-browser', gulp.series(['build-node', 'pack-browser'])
```

Reply:
I just changed the other above before pushing. And as far as I can see this isn't used at all.

Reply:
I'm also fine with removing it completely :P